reindex-discovery.py

```diff
@@ -52,7 +52,7 @@ def main():
     parser.add_option("--item_from", dest="item_from", help="starting item_id for process")
     parser.add_option("--item_to", dest="item_to", help="ending item_id for process")
     (options, args) = parser.parse_args()
-        
+    sql = "select item_id from item where owning_collection = 2 and in_archive = 't' order by item_id asc"    
```

Comment:
If you want to process most recent items, you can just set --item_to to a fairly recent number.